src/hooks/engine.rs

```diff
@@ -536,6 +536,24 @@ pub static R_DrawSkyBox: Pointer<unsafe extern "C" fn()> = Pointer::empty_patter
     ]),
     my_R_DrawSkyBox as _,
 );
+pub static R_DrawViewModel: Pointer<unsafe extern "C" fn()> = Pointer::empty_patterns(
+    b"R_DrawViewModel\0",
+    Patterns(&[
```

Comment:
Every pointer with patterns must have a "// To find" comment, could you please add them according to what Smiley wrote? In the same style as other "// To find" commends.